cmd/api-errors.go

```diff
@@ -1656,7 +1656,7 @@ func toAPIErrorCode(ctx context.Context, err error) (apiErr APIErrorCode) {
 	case UnsupportedMetadata:
 		apiErr = ErrUnsupportedMetadata
 	case BucketPolicyNotFound:
-		apiErr = ErrNoSuchBucketPolicy
```

Comment:
I guess you wanted to modify "NoSuchBucketLifecycle/BucketLifecycleNotFound", but anyways it is better to create a new one as @harshavardhana commented.